dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpAddressState.scala

```diff
@@ -0,0 +1,34 @@
+package com.wavesplatform.dex.api.http.entities
+
+import com.wavesplatform.dex.actors.address.AddressActor.Reply.GetState
+import com.wavesplatform.dex.domain.asset.Asset
+import io.swagger.annotations.ApiModelProperty
+import play.api.libs.json.{Format, Json}
+
+case class HttpAddressState(
+  @ApiModelProperty regular: Map[Asset, Long],
+  @ApiModelProperty reserved: Map[Asset, Long],
+  @ApiModelProperty allTradableBalance: Map[Asset, Long],
+  @ApiModelProperty unconfirmed: Map[Asset, Long],
+  @ApiModelProperty outgoingLeasing: Long,
+  @ApiModelProperty notCreatedTxs: Map[String, Map[Asset, Long]],
+  @ApiModelProperty notObservedTxs: Map[String, Map[Asset, Long]],
+  @ApiModelProperty placementQueue: List[String]
```

Comment:
`Order.Id` is fine here, As I remember we have a `Format` for our `ByteStr`, try to find it and import here

Reply:
ok